flow-test-util/src/main/java/com/vaadin/flow/testutil/ChromeDriverLocator.java

```diff
@@ -87,6 +103,28 @@ private static String getDriverLocation() {
 
     }
 
+    private static String getDriverFromPath() {
+        try {
+            Process p = new ProcessBuilder().command("which", "chromedriver")
```

Comment:
This should take into account that windows has `where`
```suggestion
            Process p = new ProcessBuilder().command(FrontendUtils.isWindows() ? "where" : "which", "chromedriver")
```